### PR TITLE
Transfer and sign at the same time in bench-tps

### PR DESCRIPTION
**Pull request by sakridge**

### CODE REVIEW

**Comment by mvines** [comment]
Please ping when this is ready for review, looks like it's still a wip.

**Comment by sakridge** [comment]
@mvines Yea, was running a bunch of experiments yesterday to try to get this to be the same peak performance as the non-overlapping one. I think i'll just add a switch so we have both..

**Comment by sakridge** [comment]
@mvines I think it's ready now. please take a look. thanks!

File: src/bin/bench-tps.rs [hunk 1]
```diff
@@ -236,6 +354,18 @@ fn main() {
                 .takes_value(true)
                 .help("address to advertise to the network"),
         )
+        .arg(
```
Comment (mvines) [inline src/bin/bench-tps.rs]: oh can you sneak in a `--sustained` here [1] as a part of this PR please.

[1] https://github.com/solana-labs/solana/blob/master/ci/testnet-deploy.sh#L165

Comment (sakridge) [reply src/bin/bench-tps.rs]: Sure

**Pull request merged**